src/stores/connectors/eth1/create.go

```diff
@@ -0,0 +1,28 @@
+package eth1
+
+import "C"
+import (
+	"context"
+
+	"github.com/consensys/quorum-key-manager/src/stores/entities"
+)
+
+func (c Connector) Create(ctx context.Context, id string, attr *entities.Attributes) (*entities.ETH1Account, error) {
+	logger := c.logger.With("id", id)
+	logger.Debug("creating ethereum account")
+
+	key, err := c.store.Create(ctx, id, eth1Algo, attr)
+	if err != nil {
+		return nil, err
+	}
+	
+	acc, err := c.db.Add(ctx, parseKey(key, attr))
```

Comment:
What happens if we lose connection to the DB? How do we handle these situations?

Reply:
Create an issue to discuss best way to deal with this situation

Reply:
https://github.com/ConsenSys/quorum-key-manager/issues/256